index.html

```diff
@@ -1815,6 +1815,32 @@ <h4>not-found</h4>
       </section>
 
     </section>
+
+      <section>
```

Comment:
- Move section before the above closing tag.
- Indent tree with 2 spaces like other sections.
- Remove extra trailing closing section tag.